library/common/jni/android_test_jni_interface.cc

```diff
@@ -10,7 +10,9 @@
 extern "C" JNIEXPORT jint JNICALL
 Java_io_envoyproxy_envoymobile_engine_AndroidJniLibrary_initialize(JNIEnv* env,
                                                                    jclass, // class
+                                                                   jobject class_loader,
```

Comment:
Since this method has a new signature, I would have expected the callers to need to change. Is there a reason they don't have to?

Reply:
Done - thanks!

Reply:
Yea, the issue is that the current signature - from before this PR - is the one that's wrong. AFAIk the only reason for why this compiles is that JNI does not enforce the existence of the implementation that matches whatever we expose in JNI header files. It's probably currently not crashing in main only because we do not use any of the passed arguments in here (or by luck).

Reply:
Bananas! :)